src/citrine/resources/data_concepts.py

```diff
@@ -485,8 +494,11 @@ def async_update(self, model: ResourceType, *,
             self.poll_async_update_job(job_id, timeout=timeout,
                                        polling_delay=polling_delay)
 
-            # That worked, nothing returned in this case
-            return None
+            # That worked, return nothing or return the object
+            if return_model:
+                return self.get(LinkByUID(scope=scope, id=id))
```

Comment:
@dleblanc since `async_update` currently returns `None`, I want to make sure that allowing it to get the resource isn't a bad idea. For example, might this `get` try to fetch the resource before the database has finished updating?

Reply:
This might be a concern with eventual consistency in Dynamo, but it's no worse than anywhere else that we're doing a read-after-write. I'm ok with it.